src/base-styles/base/formTags.css

```diff
@@ -84,6 +88,15 @@ select {
   height: var(--space-double);
 }
 
+/* Safari Only */
+@media not all and (min-resolution: 0.001dpcm) {
+  @media {
+    select {
+      outline: initial;
```

Comment:
fixes a safari bug where `box-shadow` doesn't show on `select` elements when `outline` is set to `0`.